test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+flake8
+stestr>=2.0.0 # Apache-2.0
+oslotest>=3.2.0 # Apache-2.0
```

Comment:
Why do you need oslotest? For a fancy base unit test class you can use `testtools`. However, you seem to be using the regular `unittest` base class right now.

Reply:
the driver test requires oslotest to work, otherwise I get ImportError while running the tests

Reply:
hmm, maybe it's pulled by oslo.messaging. okay.